zingo-sync/src/sync.rs

```diff
@@ -57,15 +62,18 @@ where
     let ufvks = wallet.get_unified_full_viewing_keys().unwrap();
     let mut scanner = Scanner::new(
         scan_results_sender,
-        fetch_request_sender,
+        fetch_request_sender.clone(),
         consensus_parameters.clone(),
-        ufvks,
+        ufvks.clone(),
     );
     scanner.spawn_workers();
 
+    // TODO: replace scanning keys with ufvk for scan_tx
+    let scanning_keys = ScanningKeys::from_account_ufvks(ufvks);
+
     let mut interval = tokio::time::interval(Duration::from_millis(30));
     loop {
-        interval.tick().await;
+        interval.tick().await; // TODO: tokio select to recieve scan results before tick
```

Comment:
Wooo! Someone other than me spelled receive wrong! I'm not alone!
```suggestion
        interval.tick().await; // TODO: tokio select to receive scan results before tick
```